eurocodedesign/core/NA.py

```diff
@@ -0,0 +1,143 @@
+"""
+National annex module
+
+This module enables the use of setting the country code from which national
+defined parameters should be used. NDP-supporting functions are annotated with
+the @NDP decorator.
+
+Example:
+    ::
+
+        >>> from eurocodedesign.core.NA import set_country
+        >>> import eurocodedesign.standard.ec3 as ec3
+        >>> ec3.gamma_M1()
+        ... 1.00
+        >>> set_country('de')
+        >>> ec3.gamma_M1()
+        ... 1.10
+        >>> ec3.gamma_M1(country=None)
+        ... 1.00
+        >>> set_country('')
+        >>> with ed.core.NA.CountryContext(country='de'):
```

Comment:
Lowercase `de or` uppercase `DE`? `NACountry` uses uppercase for the `Enum.name` and `Enum.value` but this example shows lowercase

Reply:
the parameter `country` in the call to `ed.core.NA.CountryContext` should be type `NACountry `and not string